src/pages/experiments/[experimentId]/data-exploration/components/hierarchical-tree/HierarchicalTree.jsx

```diff
@@ -220,24 +233,6 @@ const HierarchicalTree = (props) => {
     />
   );
 
-  const renderHideButton = (modified) => {
-    if (!modified.rootNode) {
-      return (
-        <Button
-          size='small'
-          onClick={(e) => {
-            e.stopPropagation();
-            propOnHide(modified.key);
-          }}
-        >
-          Hide
-        </Button>
-      );
-    }
-
-    return <></>;
-  };
-
```

Comment:
why is this deleted?

Reply:
leave it, since we will be using it to implement one of the future features of multiple-sample support